.env.example

```
@@ -9,6 +13,9 @@ IMESSAGE_SCHEDULER_PERSIST_PATH=./data/imessage-scheduler.json
 HEARTBEAT_INTERVAL_MS=3600000
 HEARTBEAT_ACTIVE_START=08:00
 HEARTBEAT_ACTIVE_END=22:00
+TURNKEY_DELEGATED_KEY_SECRET_NAMESPACE=turnkey/delegated
+HYPERLIQUID_NETWORK=mainnet
+MULTI_USER_MODE=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MULTI_USER_MODE key should go before the TURNKEY_DELEGATED_KEY_SECRET_NAMESPACE key

(UnorderedKey)